src/hyperfine/types.rs

```diff
@@ -113,6 +113,9 @@ pub enum OutputStyleOption {
 
     /// Keep coloring, but use no progress bar
     Color,
+
+    /// Disable all the output
+    None,
```

Comment:
`None` is somehow associated with `Option<…>`. Maybe rename this to `Disabled`?

Reply:
done, good point